lib/middlewares/README.md

```diff
@@ -0,0 +1,52 @@
+# Middlewares
+## Hapi
+Works with [Hapijs](https://hapijs.com/) >= v17
+### usage
+First install the dependencies in your app:
+```bash
+yarn add hapi inert agenda
```

Comment:
Although the project uses `yarn`, let's use `npm` in readmes since that's what majority of people use.